organization/eu-central-1/app-runner/config/terragrunt.hcl

```
@@ -0,0 +1,43 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+
+include "env" {
+  path = "${get_terragrunt_dir()}/../../../_common/app_runner.hcl"
+}
+
+inputs = {
+
+  service_name                     = "remote-planning-poker"
+  create_custom_domain_association = true
+  domain_name                      = "estimation.xayn.com"
+  enable_www_subdomain             = false
```

shouldn't that be true?
```suggestion
  enable_www_subdomain             = true
```

---

Yes, I'm sorry it was true but when I submitted it. I changed by mistake.